src/include/execution/sql/aggregators.h

```diff
@@ -373,14 +373,11 @@ class TopKAggregate {
   static_assert(std::is_base_of_v<Val, T>, "Template type must subclass value");
   using CppType = decltype(T::val_);
 
-  static constexpr size_t K = 16;
-  static constexpr uint64_t WIDTH = 64;
-
  public:
   /**
    * Constructor.
    */
-  TopKAggregate() : top_k_(K, WIDTH) {}
+  TopKAggregate() : top_k_() {}
```

Comment:
We do need to know K and width, I just pushed the default values down into the `TopKElements` class directly instead of this class because I thought it was more relevant to the `TopKElements` class than the aggregator wrapping it.